include/uprotocol-cpp/uri/serializer/MicroUriSerializer.h

```diff
@@ -148,7 +148,7 @@ class MicroUriSerializer {
       * @param resource 
       * @return 
       */
-    [[nodiscard]] [[maybe_unused]] auto isMicroForm(const uprotocol::v1::UEntity &entity) -> bool {
+    [[nodiscard]] [[maybe_unused]] static auto isMicroForm(const uprotocol::v1::UEntity &entity) -> bool {
         return entity.has_id() && entity.id() > 0;
```

Comment:
Please enclose logical operation in separate parenthesis.  Many other instances of this in this file also.

Reply:
Done